packages/frontend/src/components/VariablesList/index.tsx

```diff
@@ -42,7 +42,7 @@ export default function VariablesList(props: VariablesListProps) {
   return (
     <List
       disablePadding
-      data-test="power-input-suggestion-group"
+      data-test="variable-suggestion-group"
```

Comment:
noob qns: in what use cases would this be used ah?

Reply:
Right now not used yet, but the name was legacy from automatstitch code when we was still using slate so im just updating to something more generic. Could be used for clicking on the variable but I don't need to yet so far